client/my-sites/importer/importing-pane.jsx

```diff
@@ -14,6 +15,61 @@ import ProgressBar from 'components/progress-bar';
 import MappingPane from './author-mapping-pane';
 import Spinner from 'components/spinner';
 
+const sum = ( a, b ) => a + b;
+
+/*
+ * The progress object comes from the API and can
+ * contain different object counts.
+ *
+ * The attachments will lead the progress because
+ * they take the longest in almost all circumstances.
+ *
+ * progressObect ~= {
+ *     post: { completed: 3, total: 12 },
+ *     comment: { completed: 0, total: 3 },
+ *     …
+ * }
+ */
+const calculateProgress = progress => {
```

Comment:
Is it possible to avoid the redux work being a blocker for launch? We really want to try and clear the path here as much as possible.